tests/sentry/integrations/slack/test_disable.py

```diff
@@ -0,0 +1,163 @@
+from datetime import datetime, timedelta
+
+import pytest
+import responses
+from django.test import override_settings
+from freezegun import freeze_time
+
+from sentry.constants import ObjectStatus
+from sentry.integrations.request_buffer import IntegrationRequestBuffer
+from sentry.integrations.slack.client import SlackClient
+from sentry.models import Integration
+from sentry.models.integrations.organization_integration import OrganizationIntegration
+from sentry.services.hybrid_cloud.integration import integration_service
+from sentry.shared_integrations.exceptions import ApiError
+from sentry.silo.base import SiloMode
```

Comment:
Going forward when you commit it should be like `git commit -m "my commit message"` rather than using `-nm` which bypasses the pre-commit hooks (which is handy for draft mode when you want to add comments and stuff). The benefit to doing that is that it tells you about [this type of failure](https://github.com/getsentry/sentry/actions/runs/5616794974/job/15219776345?pr=52994) before you commit so you can clean up your imports and whatnot.